docs/api/operators.md

```diff
@@ -1,9 +1,3 @@
-::: oodeel.utils.operator
-    options:
-        show_root_toc_entry: True
-        inherited_members: True
-        show_submodules: True
-
 ::: oodeel.utils.tf_operator
```

Comment:
There are already tf_operator.md and torch_operator.md. Maybe we can keep here the abstract operator and remove the torch and tf operators?

Reply:
I prefer Tf operator and Torch operator to be included in the doc to show how the methods are implemented (underlying tf or torch syntax)